### PR TITLE
Add support for uncompressed .MTREEs

### PR DESCRIPTION
**Pull request by tallero**
As the title says.

### CODE REVIEW

**Comment by Edenhofer** [comment]
Many thanks for this contribution. The changes certainly look very reasonable and I am happy to merge them. However, I don't want to change the license just for this. Are you fine with sticking to GPL?

**Comment by tallero** [comment]
@Edenhofer yes, no problem!

File: fakepkg [hunk 1]
```diff
@@ -31,6 +31,8 @@ function fakebuild() {
 	PKG_NAME="${1}-${PKG_VER}"
 	PKG_DB_ENTRY="${ROOT_DIR:-/}${db_path}/local/${PKG_NAME}"
 	PKGEXT="${PKGEXT:-.pkg.tar.zst}" # implicitly sets the compression format
+        local _MTREE="${PKG_DB_ENTRY#/}/mtree"
```
Comment (Edenhofer) [inline fakepkg:34]: I am currently in the process of merging this. Can you elaborate a little as to why the replacement of the first forward slash is necessary (`#/`)?

**Comment by Edenhofer** [comment]
You may find my current version of your MR at tentative_tallero_merge (67ef2a8). I hope my changes are fine with you. If so I would be glad if you could briefly confirm this :)

File: fakepkg (hunk 1 above)
Comment (tallero) [reply fakepkg:34]: I don't remember :laughing:

I should re-install the old version to check.

**Comment by Edenhofer** [comment]
Merged in v1.42.1 .

**Pull request closed without merging**